pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: my_target_flutter
 description: Implementation MyTarget ads SDK.
-version: 0.1.0+1
+version: 0.1.0+2
```

Изменения достаточно большие, в том числе несовместимые (конструктор изменился), поэтому надо поднимать минорную версию 
```suggestion
version: 0.2.0
```

И надо добавить строчку в Changelog